Cargo.toml

```diff
@@ -8,16 +8,21 @@ repository = "https://github.com/sfackler/rust-unix-socket"
 documentation = "https://sfackler.github.io/rust-unix-socket/doc/v0.4.5/unix_socket"
 readme = "README.md"
 keywords = ["posix", "unix", "socket", "domain"]
+build = "build.rs"
 
 [dependencies]
 libc = "0.1"
 debug-builders = "0.1"
 
+[build-dependencies]
+gcc = "0.3"
+
 [dev-dependencies]
 tempdir = "0.3"
 
 [features]
-default = ["from_raw_fd"]
+default = ["from_raw_fd", "sendmsg"]
```

Comment:
Well, the SCM_RIGHTS stuff isn't portable past Linux (POSIX doesn't define what gets passed in the message, I believe)